includes/api/class-wc-admin-rest-reports-performance-indicators-controller.php

```diff
@@ -0,0 +1,268 @@
+	 * @return array|WP_Error
+	 */
+	public function get_items( $request ) {
+		$allowed_stats = $this->get_allowed_stats();
+
+		if ( is_wp_error( $allowed_stats ) ) {
+			return $allowed_stats;
+		}
+
+		$query_args = $this->prepare_reports_query( $request );
+		if ( empty( $query_args['stats'] ) ) {
+			return new WP_Error( 'woocommerce_reports_performance_indicators_empty_query', __( 'A list of stats to query must be provided.', 'wc-admin' ), 400 );
+		}
+
+		$stats = array();
+		foreach ( $query_args['stats'] as $stat_request ) {
+			$is_error = false;
+
+			$pieces   = explode( '/', $stat_request );
+			$endpoint = $pieces[0];
+			$stat     = $pieces[1];
+
+			if ( ! in_array( $stat_request, $allowed_stats ) ) {
+				continue;
+			}
+
+			$request = new WP_REST_Request( 'GET', '/wc/v3/reports/' . $endpoint . '/stats' );
+			$request->set_param( 'before', $query_args['before'] );
+			$request->set_param( 'after', $query_args['after'] );
+
+			$response = rest_do_request( $request );
+			$data     = $response->get_data();
+
+			if ( 200 !== $response->get_status() || empty( $data['totals'][ $stat ] ) ) {
+				$stats[] = (object) array(
+					'stat'  => $stat_request,
+					'value' => null,
+				);
+				continue;
+			}
+
+			$stats[] = (object) array(
+				'stat'  => $stat_request,
+				'value' => $data['totals'][ $stat ],
+			);
+		}
+
+		$objects = array();
+		foreach ( $stats as $stat ) {
+			$data      = $this->prepare_item_for_response( $stat, $request );
+			$objects[] = $this->prepare_response_for_collection( $data );
+		}
+
+		$response = rest_ensure_response( $objects );
+		$response->header( 'X-WP-Total', count( $stats ) );
+		$response->header( 'X-WP-TotalPages', 1 );
+
+		$base = add_query_arg( $request->get_query_params(), rest_url( sprintf( '/%s/%s', $this->namespace, $this->rest_base ) ) );
+
+		return $response;
+	}
+
+	/**
+	 * Prepare a report object for serialization.
+	 *
+	 * @param stdClass        $stat_data    Report data.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response
+	 */
+	public function prepare_item_for_response( $stat_data, $request ) {
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$data    = $this->add_additional_fields_to_object( $stat_data, $request );
+		$data    = $this->filter_response_by_context( $data, $context );
+
+		// Wrap the data in a response object.
+		$response = rest_ensure_response( $data );
+
+		$response->add_links( $this->prepare_links( $data ) );
+
+		/**
+		 * Filter a report returned from the API.
+		 *
+		 * Allows modification of the report data right before it is returned.
+		 *
+		 * @param WP_REST_Response $response The response object.
+		 * @param object           $report   The original report object.
+		 * @param WP_REST_Request  $request  Request used to generate the response.
+		 */
+		return apply_filters( 'woocommerce_rest_prepare_report_performance_indicators', $response, $stat_data, $request );
```

Comment:
Sometimes I should read all the code prior to commenting. This seems like it will be sufficient to allow for extending this endpoint with stats support from other sources in extensions. 🌟